packages/design-system/src/scalars/components/fragments/toggle-field/schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+
+export const formSchema = z
+  .object({
+    checked: z.boolean().optional(),
+    required: z.boolean().optional(),
```

Comment:
we should validate the `checked` property only, the `required` property would be acting as a modifier of the validation of the toggle value

Reply:
you are right, this will close , because we move to different path for validation